packages/pds/src/config/config.ts

```diff
@@ -175,10 +175,20 @@ export const envToCfg = (env: ServerEnvironment): ServerConfig => {
   const bskyAppViewCfg: ServerConfig['bskyAppView'] = {
     url: env.bskyAppViewUrl,
     did: env.bskyAppViewDid,
-    proxyModeration: env.bskyAppViewModeration ?? false,
     cdnUrlPattern: env.bskyAppViewCdnUrlPattern,
   }
 
+  if (!env.modServiceUrl) {
+    throw new Error('Must configure PDS_MOD_SERVICE_URL')
+  } else if (!env.modServiceDid) {
+    throw new Error('Must configure PDS_MOD_SERVICE_DID')
+  }
```

Comment:
Really doesn't matter a ton, but I find that `assert()` is convenient for this kind of thing.

Reply:
nice yup 👍